### PR TITLE
Adding MD5 perl module as part of install base

### PR DESCRIPTION
**Pull request by mtunganati**

### CODE REVIEW

**Comment by oneops-admin** [comment]
Build 137

File: components/cookbooks/compute/files/default/install_base.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ if [ -e /etc/SuSE-release ] ; then
 # redhat / centos
 elif [ -e /etc/redhat-release ] ; then
   echo "installing ruby, libs, headers and gcc"
-  yum -d0 -e0 -y install sudo file make gcc gcc-c++ glibc-devel libgcc ruby ruby-libs ruby-devel libxml2-devel libxslt-devel ruby-rdoc rubygems perl nagios nagios-devel nagios-plugins
+  yum -d0 -e0 -y install sudo file make gcc gcc-c++ glibc-devel libgcc ruby ruby-libs ruby-devel libxml2-devel libxslt-devel ruby-rdoc rubygems perl perl-Digest-MD5 nagios nagios-devel nagios-plugins
```
Comment (sureshg) [inline components/cookbooks/compute/files/default/install_base.sh]: @mtunganati why we need this perl module just for RHEL?

**Comment by oneops-admin** [comment]
Build 138

**Comment by oneops-admin** [comment]
Build 139

**Comment by oneops-admin** [comment]
Build 140

**Comment by oneops-admin** [comment]
Build 141

**Comment by sureshg** [comment]
`perl-Digest-MD5` is required for nagios check_log script and is not available on centos 7.

**Comment by oneops-admin** [comment]
Build 142

**Comment by oneops-admin** [comment]
Build 143

**Comment by oneops-admin** [comment]
Build 144

**Comment by oneops-admin** [comment]
Build 145

**Comment by oneops-admin** [comment]
Build 146

**Comment by oneops-admin** [comment]
Build 147

**Comment by oneops-admin** [comment]
Build 148

**Comment by oneops-admin** [comment]
Build 149

**Comment by oneops-admin** [comment]
Build 150

**Comment by oneops-admin** [comment]
Build 151

**Comment by oneops-admin** [comment]
Build 152

**Comment by oneops-admin** [comment]
Build 153

**Comment by oneops-admin** [comment]
Build 154

**Comment by oneops-admin** [comment]
Build 155

**Comment by oneops-admin** [comment]
Build 156

**Comment by oneops-admin** [comment]
Build 157

**Comment by oneops-admin** [comment]
Build 158

**Comment by oneops-admin** [comment]
Build 159

**Comment by oneops-admin** [comment]
Build 160

**Comment by oneops-admin** [comment]
Build 161

**Comment by oneops-admin** [comment]
Build 162

**Comment by oneops-admin** [comment]
Build 163

**Comment by oneops-admin** [comment]
Build 164

**Comment by oneops-admin** [comment]
Build 165

**Comment by oneops-admin** [comment]
Build 166

**Comment by oneops-admin** [comment]
Build 167

**Comment by oneops-admin** [comment]
Build 168

**Comment by oneops-admin** [comment]
Build 169

**Comment by oneops-admin** [comment]
Build 170

**Comment by oneops-admin** [comment]
Build 171

**Comment by oneops-admin** [comment]
Build 172

**Comment by oneops-admin** [comment]
Build 173

**Comment by oneops-admin** [comment]
Build 174

**Comment by oneops-admin** [comment]
Build 175

**Comment by oneops-admin** [comment]
Build 176

**Comment by oneops-admin** [comment]
Build 177

**Comment by oneops-admin** [comment]
Build 178

**Comment by oneops-admin** [comment]
Build 179

**Comment by oneops-admin** [comment]
Build 180

**Comment by oneops-admin** [comment]
Build 181

**Comment by oneops-admin** [comment]
Build 182

**Comment by oneops-admin** [comment]
Build 183

**Comment by oneops-admin** [comment]
Build 184

**Comment by oneops-admin** [comment]
Build 185

**Comment by oneops-admin** [comment]
Build 186

**Comment by oneops-admin** [comment]
Build 187

**Comment by oneops-admin** [comment]
Build 188

**Comment by oneops-admin** [comment]
Build 190

**Comment by oneops-admin** [comment]
Build 191

**Comment by oneops-admin** [comment]
Build 192

**Comment by oneops-admin** [comment]
Build 193

**Comment by oneops-admin** [comment]
Build 194

**Comment by oneops-admin** [comment]
Build 195

**Comment by oneops-admin** [comment]
Build 196

**Comment by oneops-admin** [comment]
Build 197

**Comment by oneops-admin** [comment]
Build 198

**Comment by oneops-admin** [comment]
Build 199

**Comment by oneops-admin** [comment]
Build 200

**Comment by oneops-admin** [comment]
Build 201

**Comment by oneops-admin** [comment]
Build 202

**Comment by oneops-admin** [comment]
Build 203

**Comment by oneops-admin** [comment]
Build 204

**Comment by oneops-admin** [comment]
Build 205

**Comment by oneops-admin** [comment]
Build 206

**Comment by oneops-admin** [comment]
Build 207

**Comment by oneops-admin** [comment]
Build 208

**Comment by oneops-admin** [comment]
Build 209

**Comment by oneops-admin** [comment]
Build 210

**Comment by oneops-admin** [comment]
Build 211

**Comment by oneops-admin** [comment]
Build 212

**Comment by oneops-admin** [comment]
Build 213

**Comment by oneops-admin** [comment]
Build 214

**Comment by oneops-admin** [comment]
Build 215

**Comment by oneops-admin** [comment]
Build 216

**Comment by oneops-admin** [comment]
Build 217

**Comment by oneops-admin** [comment]
Build 218

**Comment by oneops-admin** [comment]
Build 219

**Comment by oneops-admin** [comment]
Build 220

**Comment by oneops-admin** [comment]
Build 221

**Comment by oneops-admin** [comment]
Build 222

**Comment by oneops-admin** [comment]
Build 223

**Pull request merged**